mindmap/static/html/mindmap_edit.html

```diff
@@ -0,0 +1,21 @@
+<div class="wrapper-comp-settings is-active editor-with-buttons" id="settings-tab">
+    <ul class="list-input settings-list">
+      <li class="field comp-setting-entry is-set">
+        <div class="wrapper-comp-setting">
+          <label class="label setting-label" for="mindmap_display_name">Display name</label>
+          <input class="input setting-input" name="mindmap_display_name" id="mindmap_display_name" value="{{display_name}}" type="text" />
+        </div>
+      </li>
+    </ul>
+
+    <div class="xblock-actions">
+      <ul>
+        <li class="action-item">
+          <a href="#" class="button action-primary save-button">Save</a>
+        </li>
+        <li class="action-item">
+          <a href="#" class="button cancel-button">Cancel</a>
+        </li>
+      </ul>
+    </div>
+  </div>
```

Comment:
don't we have to add the cdn here too?

Reply:
No, sorry. In the studio view

Reply:
🤔It should only be added in the `mindmap.js` that is loaded in the `student_view`.